helm/nats-streaming-server/templates/deployment.yaml

```diff
@@ -6,9 +6,22 @@ metadata:
 data:
   # Used to set cluster id in deployment
   # Look for $(NATS_CLUSTER) usage below
-  NATS_CLUSTER: nats-streaming
+  NATS_CLUSTER: {{ $.Values.tenantId | default "mds" }}
   # This is the NATS hostname to connect to
-  NATS_HOST: nats://nats:4222
+  NATS_HOST: nats://{{ default "nats" $.Values.natsNamespace }}-nats-server.{{ default "nats" $.Values.natsNamespace }}.svc.cluster.local:4222
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: stan-config
+  namespace: {{ .Release.Namespace }}
+data:
+  stan.conf: |
+    streaming {
+{{- if false }}
+      credentials: "/var/run/nats/secrets/sys.creds"
+{{- end }}
+    }
```

Comment:
Removeable

Reply:
required for surveyor

Reply:
this helm chart should be removed ... looking into it